api/src/API/Controllers/RaceResultsController.cs

```diff
@@ -46,7 +46,24 @@ public async Task<IActionResult> GetAllRaceResults(string orgId)
             IEnumerable<string> memberIds = members.Values.Select(member => member.Id.ToString());
 
             RaceResultContainerClient raceResultContainer = containerProvider.RaceResultContainer;
-            IEnumerable<RaceResult> raceResults = await raceResultContainer.GetRaceResultsForMembersAsync(memberIds);
+
+            string startDateParam = Request.Query["startDate"];
+            string endDateParam = Request.Query["endDate"];
+
+            DateTime? startDate = null;
```

Comment:
My understanding is that if we use model binding (see above), type validation will happen automatically. Same as when we bind to the request body. If the datetime isn't parsable, asp.net will automatically respond with a 400.

Reply:
type validation is now being done with `FromQuery` attributes on the route. I made them default to `null`, and then provide sensible defaults in the beginning of the method (`DateTime.MinValue` and `DateTime.MaxValue`).